uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/ThemeNameEqualsIgnoreCaseTester.java

```diff
@@ -48,6 +51,21 @@
     private final IUserLayoutStore userLayoutStore;
     private final IStylesheetDescriptorDao stylesheetDescriptorDao;
 
+    public ThemeNameEqualsIgnoreCaseTester(IPersonAttributesGroupTestDefinition definition) {
+        final String themeTestValue = definition.getTestValue();
+        assert StringUtils.isNotBlank(themeTestValue);
+        this.themeTestValue = themeTestValue;
+        this.applicationContext = ApplicationContextLocator.getApplicationContext();
+        this.portalRequestUtils = applicationContext.getBean(IPortalRequestUtils.class);
+        this.profileMapper = applicationContext.getBean("profileMapper", IProfileMapper.class);
+        this.userLayoutStore = applicationContext.getBean("userLayoutStore", IUserLayoutStore.class);
+        this.stylesheetDescriptorDao = applicationContext.getBean("stylesheetDescriptorDao", IStylesheetDescriptorDao.class);
+    }
```

Comment:
I think the easiest thing to do is have ThemeNameEqualsIgnoreCaseTester(IPersonAttributesGroupTestDefinition definition) invoke ThemeNameEqualsIgnoreCaseTester(null, definition.getTestValue())